examples/automatic_speech_recognition/automatic_speech_recognition/utils.py

```diff
@@ -0,0 +1,87 @@
+import re
+import difflib
+from numwords_to_nums.numwords_to_nums import NumWordsToNum
+
+from typing import Union
+
+from workflow import GroundTruth
+from workflow import Inference
+
+def generate_diff_word_level(reference: str, candidate: str, mode: str):
```

Comment:
should we use `ground_truth` and `inference` here instead of `reference` and `candidate`?